### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by georgewood749**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -5,44 +5,120 @@
     <meta charset="UTF-8">
```
Comment (Brendan-Geoghegan) [inline client/index.html:5]: Functionality is really cool, the fact that you can search any key word and results with that keyword appear is very cool. Definitely going to try and implement that in ours.

File: client/index2.html [hunk 2]
```diff
@@ -76,6 +76,7 @@
         </div>
```
Comment (Brendan-Geoghegan) [inline client/index2.html]: Index2 looks really cool and when the functionality is added it will look very professional

Comment (Brendan-Geoghegan) [reply client/index2.html]: Favicon isn't working for some reason.

File: client/index.html [hunk 3]
```diff
@@ -5,44 +5,120 @@
...
-
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8"
         crossorigin="anonymous"></script>
-
-    <title>Woogle</title>
-    <link rel="shortcut icon" href="./woogle.png" type="image/x-icon">
+    <link rel="shortcut icon" href="../client/images/woogle.png" type="image/x-icon">
```
Comment (GimmeDaLute) [inline client/index.html]: Woogle.png doesn't render on index.html unfortunately

File: client/script.js [hunk 4]
```diff
@@ -1,8 +1,13 @@
+//Set up Blind Event Listeners
 document.getElementById(`searchBar`).addEventListener('submit',search);
 document.getElementById('feelingLucky').addEventListener('click',getRandom);
 document.getElementById('showMeAll').addEventListener('click',getAll);
 document.getElementById('showSponsors').addEventListener('click',getSponsors);
 
+//Initialised Variables
```
Comment (Brendan-Geoghegan) [inline client/script.js]: These variables are not in your final push

File: server/data.js [hunk 5]
```diff
@@ -65,55 +65,55 @@ const data = [
...
     {   id: 107,
         url: "https://github.com/",
-        title: "GitHub: Where the world builds software · GitHub",
+        title: "(!SPONSORS!) GitHub: Where the world builds software · GitHub",
         des: "GitHub is where over 83 million developers shape the future of software, together. Contribute to the open source community, manage your Git repositories, ..."
     },
     {   id: 108,
         url: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
-        title: "Important stuff here",
+        title: "(!SPONSORS!) Important stuff here",
```
Comment (GimmeDaLute) [inline server/data.js:106]: https://copypastatext.com/tag/rick-roll-ascii-art

File: client/style.css [hunk 6]
```diff
@@ -1,66 +1,186 @@
-footer{
```
Comment (Brendan-Geoghegan) [inline client/style.css:1]: All css looks really cool. Looks just like google

File: server/data.js [hunk 7]
```diff
@@ -65,55 +65,55 @@ const data = [
     //10 sponser results
     {   id: 101,
         url: "https://www.cat.com/",
-        title: "Cat.com | Cat | Caterpillar",
+        title: "(!SPONSORS!) Cat.com | Cat | Caterpillar",
```
Comment (Brendan-Geoghegan) [inline server/data.js:68]: Sponsors look really  cool and work, but can't see the "(!SPONSER)" Title in final push.

File: client/index.html [hunk 8]
```diff
@@ -5,44 +5,121 @@
     <meta charset="UTF-8">
```
Comment (GimmeDaLute) [inline client/index.html:5]: README.md missing

File: client/index.html (hunk 3 above)
Comment (georgewood749) [reply client/index.html]: Ah yeah, need to update that, I did some tweaking on the logo and adjusted the file structure a few times lol

File: client/index_old.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (Brendan-Geoghegan) [inline client/index_old.html]: We can only see your last 10 commits on this review so we can't comment on all the other changes at the correct places but I just put some additional ones here about the others parts not in these commits

File: server/data.js (hunk 5 above)
Comment (georgewood749) [reply server/data.js:106]: This is brilliant! 😂😂😂

File: client/style.css (hunk 6 above)
Comment (georgewood749) [reply client/style.css:1]: Ngl I spent far too long trying to ape it 😂

File: client/index_old.html (hunk 9 above)
Comment (Brendan-Geoghegan) [reply client/index_old.html]: the filter function that you created is awesome, it literally does most of what several of my own functions try (and don't do as well) to do so definitely going to try implement it. All I would say is change the name so it doesn't get confused with the built-in filter function

File: client/index.html (hunk 1 above)
Comment (georgewood749) [reply client/index.html:5]: Yeah Kai's done a sick job with it, we're thinking of displaying the results on a new page, but may be more trouble than its worth haha

File: client/index_old.html (hunk 9 above)
Comment (Brendan-Geoghegan) [reply client/index_old.html]: Functions in the final commit for the script.js are all well displayed and easily readable. You can follow through what each one does quite well

Comment (Brendan-Geoghegan) [reply client/index_old.html]: The RegExp function looks very cool, I'm definitely going to look that one up

File: client/index2.html (hunk 2 above)
Comment (georgewood749) [reply client/index2.html]: Yeah I'll get on that in a moment, I decided to go for a different image anyway

File: client/index.html (hunk 8 above)
Comment (georgewood749) [reply client/index.html:5]: Should be in the server directory, but there's not really anything in it yet anyway lol

File: client/index_old.html (hunk 9 above)
Comment (georgewood749) [reply client/index_old.html]: Thanks for all the comments guys! We're gonna carry on tweaking the project today and work on the issues you mentioned, and hopefully add some functionality and perfect the styling!

**Pull request merged**